server/room.js

```diff
@@ -1,5 +1,118 @@
 var _ = require('underscore')._;
-const { uniqueNamesGenerator, adjectives, colors, animals } = require('unique-names-generator');
+const { uniqueNamesGenerator, adjectives, animals } = require('unique-names-generator');
+
+const colours = [
+  '#144272',
+  '#232D3F',
+  '#2D3250',
+  '#30475E',
+  '#46C2CB',
+  '#A2678A',
+  '#BE3144',
+  'black',
+  'blueviolet',
+  'brown',
+  'cadetblue',
+  'chocolate',
+  'coral',
+  'crimson',
+  'darkblue',
+  'darkcyan',
+  'darkgoldenrod',
+  'darkgreen',
+  'darkkhaki',
+  'darkmagenta',
+  'darkolivegreen',
+  'darkorange',
+  'darkorchid',
+  'darkseagreen',
+  'darkslateblue',
+  'darkslategrey',
+  'darkviolet',
+  'deeppink',
+  'dodgerblue',
+  'firebrick',
+  'forestgreen',
+  'goldenrod',
+  'green',
+  'hotpink',
+  'indianred',
+  'indigo',
+  'lightsalmon',
+  'lightseagreen',
+  'magenta',
+  'maroon',
+  'mediumblue',
+  'mediumpurple',
+  'mediumseagreen',
+  'mediumslateblue',
+  'mediumvioletred',
+  'midnightblue',
+  'navy',
+  'olive',
+  'olivedrab',
+  'orangered',
+  'palevioletred',
+  'peru',
+  'purple',
+  'rebeccapurle',
+  'red',
+  'royalblue',
+  'saddlebrown',
+  'salmon',
+  'steelblue',
+  'teal',
+  'yellowgreen'
+];
+
+const people_roles_and_objects = [
+  'android',
+  'avenger',
+  'bindelstick',
+  'blue eyes white dragon',
+  'boba fett',
+  'bulbasaur',
+  'captain janeway',
+  'captain picard',
+  'card picker',
+  'charmander',
+  'chocobo',
+  'commander sisko',
+  'deana troi',
+  'director',
+  'dobby',
+  'droid',
+  'engineer',
+  'exodia',
+  'geordi laforge',
+  'han solo',
+  'hari seldon',
+  'hypersphere',
+  'jedi',
+  'luxanna troi',
+  'mandalorian',
+  'millenium puzzle',
+  'oddish',
+  'pikachu',
+  'pingu',
+  'plant pot',
+  'poet',
+  'riker',
```

Comment:
```suggestion
  'riker',
  'riker transporter clone (thomas)',
```

Reply:
I appreciate this but the card is only so big, I will however add this as a consolation:

```suggestion
  'riker',
  'transporter clone',
  'spock',
```